guide/14-deep-learning/inference-only-text-models-in-arcgis-learn.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<p>Transformers in <strong>Natural Language Processing (NLP)</strong> are novel architectures that aims to solve sequence-to-sequence tasks while handling long-range dependencies with ease. The transformers are the latest and advanced models that give state of the art results for a wide range of tasks such as <strong>text/sequence classification</strong>, <strong>named entity recognition (ner)</strong>, <strong>question answering</strong>, <strong>machine translation</strong>, <strong>text summarization</strong>, <strong>text generation</strong> etc.</p><p> </p><p>Transformers in <strong>Natural Language Processing (NLP)</strong> are novel architectures that aim <s>aims</s> to solve sequence-to-sequence tasks while handling long-range dependencies with ease. The transformers are the latest and advanced models that give state of the art results for a wide range of tasks such as <strong>text/sequence classification</strong>, <strong>named entity recognition (<s>ner</s>NER)</strong>, <strong>question answering</strong>, <strong>machine translation</strong>, <strong>text summarization</strong>, <strong>text generation</strong> etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Made the necessary changes</p>